### ISSUE TITLE
Recovery from 2.6.3 mount bug

### ISSUE THREAD

**Issue by kapolos** [rancher/rancher#36715]
I've probably been bitten by the 2.6.3 bug (I use `volumes--from`) described on the release notes, but with a small twist.

When I had upgraded to 2.6.3, it actually worked fine for a while. At some later point though, the container would keep restarting. The logs were mentioning something about a file that already existed. Then I saw the workaround instructions but the "new" directory already existed and it had contents (so whatever migration probably happened on its own). The I tried the patch-1 version but nothing. Now the container simply restarts every 6 seconds without any logs.

My question is, what is the proper way to get a workable rancher with the existing cluster it was used to manage? That cluster was created by Rancher.

1. Can I just start a new Rancher with different volume maps and attach the existing cluster? My concern was that in the [docs](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/registered-clusters/#registering-a-cluster), it says `You can not re-register a cluster that is currently active in a Rancher setup.`. Is my Rancher created cluster now consider "active in a Rancher setup", now that the Rancher container is dead?

2. Can I use a backup of `/var/lib/rancher` from 2.5.11 and launch 2.5.11 and it will just work? Or will a mess ensue? I don't remember if I had upgraded the K8s version after I switched to 2.6.x (if it matters).

Please advise.

**Comment by barstown** [comment]
I see you marked this as closed, how were you able to resolve your issue in this case? For me, this install of 2.6.3 was the initial install on a new environment so I don't have any older version to roll back to, but I'm suddenly in a position where it's restarting constantly. I just tried to update to patch1 which didn't help. You mention a bug and a workaround, but I haven't been able to find any info on that. Are you able to share?

**Comment by kapolos** [comment]
@barstown I restored the 5.2.11 backup but it didn't work. Then I restored the 5.2.8 backup (my oldest one) and it was stopping due to a certificate being expired - then I changed the server date to the past, restarted it and it worked. Then I switched to the 5.2.11 version and it was throwing a ton of messages in the logs, but worked. Finally, I switched to the patch-1 version and everything worked as expected without the logs complaining.

This is the [bug](https://github.com/rancher/rancher/issues/35955) and this was the [workaround](https://github.com/rancher/rancher/issues/35955#issuecomment-1001016820) that didn't work for me.
